using-timescaledb/reading-data.md

```diff
@@ -147,6 +147,26 @@ SELECT
   ORDER BY time
 ```
 
+### Delta [](rate)
+
+In many monitoring (and IoT) use cases, devices or sensors report metrics that typically do not
+change. In cases where the value changes, these changes are considered anomalies. When querying
+for these changes in values over time, users typically do not want to pull all the values where
```

Comment:
i think transmit is the better verb instead of pull, its unclear what pull refers to , the queries still scans the rows but they are not transmitted to the client

Reply:
Done.